cni-plugin/pkg/k8s/k8s.go

```diff
@@ -227,8 +228,12 @@ func CmdAddK8s(ctx context.Context, args *skel.CmdArgs, conf types.NetConf, epID
 			if len(v6poolpod) != 0 {
 				v6pools = v6poolpod
 			}
+			ipFamiliesPod := annot["cni.projectcalico.org/ipFamilies"]
```

Comment:
What do you think about making this symmetrical with the Services API, which has an identical concept?

> You can set .spec.ipFamilies to any of the following array values:
> ["IPv4"]
> ["IPv6"]
> ["IPv4","IPv6"] (dual stack)
> ["IPv6","IPv4"] (dual stack)

e.g.,

```
cni.projectcalico.org/ipFamilies: '["IPv4"]'
```

Reply:
> What do you think about making this symmetrical with the Services API, which has an identical concept?
> 
> > You can set .spec.ipFamilies to any of the following array values:
> > ["IPv4"]
> > ["IPv6"]
> > ["IPv4","IPv6"] (dual stack)
> > ["IPv6","IPv4"] (dual stack)
> 
> e.g.,
> 
> ```
> cni.projectcalico.org/ipFamilies: '["IPv4"]'
> ```

I agree. Making it symmetrical with the Services API makes sense.